RNTester/js/examples/PlatformColor/PlatformColorExample.js

```diff
@@ -22,94 +22,94 @@ function PlatformColorsExample() {
       colors = [
         // https://developer.apple.com/documentation/uikit/uicolor/ui_element_colors
         // Label Colors
-        {label: 'labelColor', color: PlatformColor('labelColor')},
```

Comment:
Do you want to keep any examples that include the Color suffix? And / or maybe have an automated test that verifies both ways work?

Reply:
As I said in the Test notes, there still are ObjC examples later in this file and in unit tests

Reply:
Ah, whoops. I should read better :D